File: 프로그래머스/lv0/120835. 진료 순서 정하기/진료 순서 정하기.py
def solution(emergency):
    answer = [0 for _ in range(len(emergency))]
    sortedEmergency = sorted(emergency[:])
    
    rank = 1
    while len(sortedEmergency):
        check = sortedEmergency.pop()
        index = emergency.index(check)
        answer[index] = rank
        rank += 1
    return answer

File: 프로그래머스/0/120835. 진료 순서 정하기/진료 순서 정하기.js
function solution(emergency) {
    var answer = Array(emergency.length).fill(0);
    const sortedEmergency = [...emergency].sort((a, b) => b - a);
    emergency.forEach((item, i) => {
        const index = sortedEmergency.indexOf(item);
        answer[i] = index + 1;
    })
    return answer;
}